nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -378,6 +381,9 @@ def extcap_capture(self, fifo, dev, channel, control_in=None, control_out=None):
         self.dev = dev
         self.running.set()
 
+        if len(self.threads):
```

Comment:
Perhaps I lack the full context, but what is the intention behind this conditional? Is this meant to protect the user from calling `extcap_capture` on errored and/or not closed sniffer object when running in non-standalone mode? I believe that this check should be done at the very beginning of this method.